programs/client/Client.cpp

```diff
@@ -1120,7 +1147,12 @@ void Client::processOptions(const OptionsDescription & options_description,
     if (options.count("config"))
         config().setString("config-file", options["config"].as<std::string>());
     if (options.count("host") && !options["host"].defaulted())
-        config().setString("host", options["host"].as<std::string>());
+    {
+        hosts_ports = options["host"].as<std::vector<HostPort>>();
+        config().setString("host", hosts_ports[0].host);
```

Comment:
Is it better to set all the hosts and ports instead of just one?

Reply:
`config` can't store `std::vector` , only `String`, `Int` and other simple types. I wrote about this in this PR: https://github.com/ManagedDatabases/ClickHouse/pull/4#discussion_r740946579

Reply:
OK